go.mod

```diff
@@ -46,6 +46,8 @@ require (
 	github.com/pelletier/go-toml v1.8.1
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/procfs v0.6.0 // indirect; temporarily force v0.6.0, which was previously defined in imgcrypt as explicit version
```

Comment:
Without this, it would downgrade to v0.2.0, which is the version that github.com/prometheus/client_golang uses in its go.mod. Perhaps we should downgrade to match that version, but I tried to keep the diff small

Reply:
Ack